audit/formatter.go

```diff
@@ -21,6 +22,7 @@ type FormatterConfig struct {
 	Raw          bool
 	Salt         *salt.Salt
 	HMACAccessor bool
+	Mode         os.FileMode
```

Comment:
Yes, a particular instantiation of the file audit backend (as in, not a package var)